pkg/pool-weighted/test/BaseWeightedPool.behavior.ts

```diff
@@ -569,15 +575,15 @@ export function itBehavesAsWeightedPool(
       context('given in', () => {
         it('reverts if caller is not the vault', async () => {
           await expect(
-            pool.instance.onSwap(
+            pool.instance[MINIMAL_SWAP_INFO_ONSWAP](
               {
                 kind: SwapKind.GivenIn,
                 tokenIn: tokens.first.address,
                 tokenOut: tokens.second.address,
                 amount: 0,
                 poolId: await pool.getPoolId(),
                 lastChangeBlock: 0,
-                from: other.address,
+                from: lp.address,
```

Comment:
Why are we using `lp` here, but not below in `given out` context?

Reply:
Only because it didn't fail, I guess. I can change it there as well for consistency.